samples/DataSourceWithProcessingAndExtensions/Sample/CustomDataSource.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Performance.SDK.Processing;
+
+namespace DataExtensionsSample
+{
+    /// <summary>
+    /// This custom data source defines the type of data it will process. In this case, a file with a "des" filename
+    /// extension. 
+    /// </summary>
+    [CustomDataSource(
```

Comment:
>CustomDataSource [](start = 5, length = 16)

How would we feel about not naming the sample CDS classes "CustomDataSource" and the like? The generic names make it feel like names from the actual SDK instead of top-level classes that the sample defines. Maybe we should name these files "DesDataSource, "DesDataProcessor", and "DesSourceParser?"